packages/builders/scripts/vats/updatePriceFeeds.js

```diff
@@ -0,0 +1,90 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+import { getBrandsAndProposalBuilder as pricefeedProposalBuilder } from '../inter-protocol/price-feed-core.js';
+
+const ORACLE_ADDRESSES = [
+  // 'agoric1cn8dwvv3ac329sujgpqmccv6xd9d3sagd2t5l9',
+  // 'agoric1dxe9dqu4ejpctnp2un7s4l7l8hny6dzr8juur3',
+  // 'agoric1y750klm8sh8yh4h4mcssx5m7rmvuu8mhk89t0c',
+
+  // XXX These are the GOV1ADDR, etc addresses. What should this file specify?
```

Comment:
`scripts/inter-protocol/price-feed-core.js` gets them from the environment and arranges for them to be passed to `defaultProposalBuilder`. or something.

Reply:
@turadg pointed out that the proposals get built before the environment that would generate addresses exists, so they have to be hard-coded or substituted or something. This relevant issue (https://github.com/Agoric/agoric-3-proposals/issues/5) is apparently coming soon.